src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -128,29 +130,26 @@ public static synchronized FirebaseAuth getInstance(FirebaseApp app) {
    */
   public Task<String> createCustomToken(
       final String uid, final Map<String, Object> developerClaims) {
-    FirebaseCredential credential = ImplFirebaseTrampolines.getCredential(firebaseApp);
-    if (!(credential instanceof FirebaseCredentials.CertCredential)) {
+    GoogleCredentials googleCredentials = ImplFirebaseTrampolines.getCredentials(firebaseApp);
+    if (!(googleCredentials instanceof ServiceAccountCredentials)) {
       return Tasks.forException(
           new FirebaseException(
-              "Must initialize FirebaseApp with a certificate credential to call "
+              "Must initialize FirebaseApp with a service account credential to call "
                   + "createCustomToken()"));
```

Comment:
Fixed. While working on this I also realized that since we are no longer getting tokens via `FirebseApp`, some `FirebaseAuth` methods can continue to work even after deleting the app. Fixed this problem too.